Libraries/Components/AccessibilityInfo/AccessibilityInfo.android.js

```diff
@@ -37,24 +40,51 @@ const _subscriptions = new Map();
 const AccessibilityInfo = {
   /* $FlowFixMe(>=0.78.0 site=react_native_android_fb) This issue was found
```

Comment:
In order to have confidence in these changes, can you investigate what it would take to have Flow pass if we remove these `$FlowFixMe`s?

Reply:
Fixed @TheSavior. It was just missing the proper definition for the `Promise`.